app/pods/agendas/template.hbs

```diff
@@ -19,12 +40,176 @@
           >
             {{t "new-agenda"}}
           </Auk::Button>
-        </Auk::Toolbar::Item>
+        </Group.Item>
       {{/if}}
-    </Auk::Toolbar::Group>
+    </Toolbar.Group>
   </Auk::Toolbar>
 </Auk::Navbar>
-{{outlet}}
+{{#if this.filter}}
+  <ChangesAlert
+    @message={{t "filtered-agendas"}}
+    @buttonText={{t "clear-filter"}}
+    @clearAction={{this.clearFilter}}
+  />
+{{/if}}
+<div class="auk-scroll-wrapper">
+  <div class="auk-scroll-wrapper__body">
+    {{#if @model.length}}
+      <div class="auk-u-ml-4">
+        <table
+          data-test-route-agendas-overview-data-table
+          class="auk-table auk-table--agenda auk-table--sticky-header"
+        >
+          <thead>
+            <tr>
+              <Utils::ThSortable
+                class="auk-table__col--3 auk-table__header-title"
+                @currentSorting={{this.sort}}
+                @field="created-for.planned-start"
+                @onChange={{this.sortTable}}
+                @label={{t "agenda"}}
+              />
+              <Utils::ThSortable
+                class="auk-table__header-title"
+                @currentSorting={{this.sort}}
+                @field="created-for.is-final"
+                @onChange={{this.sortTable}}
+                @label={{t "status"}}
+              />
+              <Utils::ThSortable
+                class="auk-table__header-title"
+                @currentSorting={{this.sort}}
+                @field="created-for.kind.label"
+                @onChange={{this.sortTable}}
+                @label={{t "kind"}}
+              />
+              <Utils::ThSortable
+                class="auk-table__header-title"
+                @currentSorting={{this.sort}}
+                @field="modified"
+                @onChange={{this.sortTable}}
+                @label={{t "latest-modified"}}
+              />
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+            {{#if this.isLoadingModel}}
+              <tr>
+                <td
+                  data-test-route-agendas-overview-loader
+                  colspan="5"
+                >
+                  {{t "loading"}}
+                </td>
+              </tr>
+            {{else}}
+              {{#each @model as |agenda|}}
+                <tr
+                  class="auk-u-text-align--center auk-table--interaction-cursor auk-table--clickable-rows"
+                  {{! template-lint-disable no-invalid-interactive }}
+                  {{on "click" (fn this.onClickRow agenda)}}
+                >
+                  <td data-test-route-agendas-overview-row-title>
+                    <span class="auk-o-flex auk-o-flex--vertical-center auk-o-flex--spaced">
+                      {{#if agenda.status.isDesignAgenda}}
+                        <strong>{{t "design-agenda"}}</strong>
+                        <Auk::Pill @skin="active-future">{{agenda.name}}</Auk::Pill>
```

Comment:
Misschien beter de non-aliased property `agenda.serialnumber` gebruiken. We moeten er rekening mee houden dat legacy agenda's geen serialnumber hebben, en we dan dus geen pill moeten tonen. 

In de collapsed view, waar enkel serialnumber getoond wordt, is er dan niets zichtbaar. Maar dat lijkt op productie nu ook het geval te zijn :thinking:  Misschien even checken met design wat we dan best tonen.

Reply:
heb het alvast aangepast naar serialnumber en met een if gewrapt, maar laat dit dan unresolved tot we gecheckt hebben met de design fellas

Reply:
Aangezien we nu al niets tonen in de collapsed view op prod zou ik deze niet laten wachten.